lib/au_core_test_kit/generator/search_definition_metadata_extractor.rb

```diff
@@ -170,12 +171,60 @@ def chain
       end
 
       def multiple_or_expectation
+        # NOTE: https://github.com/hl7au/au-fhir-core-inferno/issues/61
```

Comment:
Add note why we have this hardcoded value, 
Links to issues is good, but I would like to get high-level context without reviewing them